app/controllers/onboardings_controller.rb

```diff
@@ -1,19 +1,15 @@
 class OnboardingsController < ApplicationController
   layout "application"
-
   before_action :set_user
-
-  def show
-  end
-
-  def profile
-  end
-
-  def preferences
-  end
```

Comment:
Can we keep these?  I know it's a bit redundant but if we've got a route for them I think having the action (even if empty) provides good documentation for future devs